src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpSamDAO.scala

```diff
@@ -22,4 +23,12 @@ class HttpSamDAO( implicit val system: ActorSystem, implicit val executionContex
     authedRequestToObject[RegistrationInfo](Get(samUserRegistrationUrl))
   }
 
+  override def status: Future[SubsystemStatus] = {
+    val pipeline = sendReceive
+    pipeline(Get(samStatusUrl)) map { response =>
```

Comment:
Same comment as in Rawls PR: https://github.com/broadinstitute/rawls/pull/796#discussion_r144183054

Reply:
any exception in this process is handled by StatusService